iron-ajax.html

```diff
@@ -357,7 +350,7 @@
       request.completes.then(
         this._boundHandleResponse
       ).catch(
-        this._boundHandleError
+        this.handleError.bind(this, request)
```

Comment:
I don't think so. They are bound in advance because they can be. It's better to bind them in advance if you can afford to do it, because you aren't creating a new bound function instance each time a request is made.

The error handler has to be bound per-request, so that the request can be properly associated with the error.